packages/components/src/ui/tooltip/test/index.js

```diff
@@ -37,14 +37,14 @@ describe( 'props', () => {
 				<Text>{ invisibleTooltipTriggerContent }</Text>
 			</Tooltip>
 		);
-		const tooltips = screen.getAllByRole( /tooltip/i );
+		const tooltip = screen.getByRole( /tooltip/i );
 		const invisibleTooltipTrigger = screen.getByText(
 			invisibleTooltipTriggerContent
 		);
-		// The invisible tooltip should not render.
-		expect( tooltips ).toHaveLength( 1 );
+		// The base tooltip should render only; invisible tooltip should not render.
```

Comment:
Nit — I'm not I understand this inline comment!

Reply:
For some reason in the original test, the "WordPress.org - Invisible" text is referred to as "invisible tooltip". I've just kept that reference. But I'm happy to improve it if you have an idea why we were referring to it as "invisible tooltip" in the first place!